### ISSUE TITLE
"return to document" should go to your previous place, not top of page

### ISSUE THREAD

**Issue by alexrudnick** [hltdi/guampa#37]
Currently on the "history" page, if you hit the "return to document" link, you go to the  top of the document. But if you were scrolled way down, you probably want to go back to the sentence you were looking at.

There are ways to make the page automatically scroll to a particular element, right? Focus?

**Comment by alexrudnick** [comment]
@SirSkidmore -- could you take this one?

**Comment by taylskid** [comment]
@alexrudnick yeah, I can try. I know it can be done, I just have to figure out how
